src/views/Questionnaire/components/RemsInterface/RemsInterface.tsx

```diff
@@ -36,42 +21,30 @@ export default function RemsInterface(props: RemsInterfaceProps) {
     return bundle.entry?.map(entry => {
```

Comment:
This seems to be causing a unique prop key warning. doesnt cause any breaking changes but may want to address this at some point 
![Screenshot 2024-05-17 at 10 59 31 AM](https://github.com/mcode/rems-smart-on-fhir/assets/28585306/f95cef3b-74a4-4776-bce7-5b71c119ad97)
![Screenshot 2024-05-17 at 10 59 35 AM](https://github.com/mcode/rems-smart-on-fhir/assets/28585306/33df1cb3-fe35-42f5-bee1-7e33f05996dd)